modules/eks/actions-runner-controller/README.md

```diff
@@ -56,16 +65,45 @@ components:
               scale_down_factor: 0.5
             resources:
               limits:
-                cpu: 50m
-                memory: 100Mi
+                cpu: 200m
+                memory: 1Gi
               requests:
-                cpu: 10m
-                memory: 30Mi
+                cpu: 100m
+                memory: 128Mi
             webhook_driven_scaling_enabled: false
```

Comment:
We want to use webhook driven scaling wherever possible.

Reply:
added